.husky/commit-msg

```diff
@@ -17,6 +17,7 @@ fi
 # Initialise git-secrets configuration
 git-secrets --register-aws > /dev/null
 
-echo "Running git-secrets..."
+echo "Checking commit message for secrets..."
 # Scans the commit message.
 git-secrets --commit_msg_hook -- "$@"
+echo "Checking commit message for secrets...DONE"
```

Comment:
Might this be a leftover of some test?

Reply:
I added this back in as I thought this was just a simple message that would help it be clearer as to what is happening.